EDSAnalytics/EDSTest/EDSAnalytics.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We have been trying to add StyleCop and FxCop analyzers to our projects, see [this project](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/Authentication/ClientCredentialFlow/DotNet) for an example. If you don't have the bandwidth to do it, we can either work on the branch to do it or go back and add that later.